website/landingpage/dev/src/assets/scss/components/_home_page.scss

```diff
@@ -52,6 +52,13 @@
 
     .description {
       margin-left: 30px;
+      a {
+        font-weight: 600;
+        color: #3c4553;
```

Comment:
Please use variables for colors. Take a look at `assets/scss/_settings.scss` file. There is a `$foundation-palette` variable.

Reply:
done